test/recipes/30-test_evp_data/evpkdf.txt

```diff
@@ -254,39 +254,39 @@ Output = 2c91117204d745f3500d636a62f64f0ab3bae548aa53d423b0d1f27ebba6f5e5673a081
 
 Title = id-scrypt tests (from draft-josefsson-id-scrypt-kdf-03 and others)
 
-KDF = id-scrypt
```

Comment:
This can not be changed until aliases are supported in providers.
It currently ises the short name which is id-scrypt.

Reply:
Then I think the check in `evp_test.c` could changed, right?

Reply:
There are a couple of if (strcmp(name, "scrypt") == 0) already - so yes it could be changed..
maybe also add /* TODO(3.0) Replace with "scrypt" once aliases are supported */